### PR TITLE
add docker test container

### PR DESCRIPTION
**Pull request by DenverM80**
will need to update the run_docker.rb script with the proper container ID after it is created

### CODE REVIEW

File: docker/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+Apache License
```
Comment (rpmoore) [inline docker/LICENSE]: Why did we add a license file here?  We already have one at the root of the repo.

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ds3-c-docker
+A public docker file that has all the dependencies to build and develop the DS3 C SDK
+
+To use this docker container, run the following command:
+
+`sudo docker run -it spectralogic/ds3_c_docker`
```
Comment (DenverM80) [inline docker/README.md]: this will be updated once we have a public container to reference

File: docker/LICENSE (hunk 1 above)
Comment (DenverM80) [inline docker/LICENSE]: I copied this from the ds3_c_docker repo, which needed it at the root leve.  removed.  (I think..?)

File: docker/run_docker.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+ENV["DS3_ENDPOINT"] = "sm25-2.eng.sldomain.com"
+ENV["DS3_SECRET_KEY"] = spectra_user_keys["secret_key"]
+ENV["DS3_ACCESS_KEY"] = spectra_user_keys["auth_id"]
+puts "DS3_ENDPOINT #{ENV["DS3_ENDPOINT"]}"
+puts "DS3_SECRET_KEY #{ENV["DS3_SECRET_KEY"]}"
+puts "DS3_ACCESS_KEY #{ENV["DS3_ACCESS_KEY"]}"
+
+puts "docker run -e DS3_ENDPOINT -e DS3_SECRET_KEY -e DS3_ACCESS_KEY -it 6ab6d85c5f45"
+output = `docker run -e DS3_ENDPOINT -e DS3_SECRET_KEY -e DS3_ACCESS_KEY -it 6ab6d85c5f45`
```
Comment (DenverM80) [inline docker/run_docker.rb]: this container ID will be updated once its public

**Pull request merged**